cmd/configure/cmd.go

```diff
@@ -88,9 +83,49 @@ func NewCommand() *cobra.Command {
 				output.Errorf("failed to request api7 cloud: %s", err)
 			}
 
+			profileName := options.Global.Configure.Profile
+
+			configuration, err := persistence.LoadConfiguration()
+			if err != nil {
+				output.Verbosef("there is no configuration file, create a new one")
+				configuration = &persistence.CloudConfiguration{}
+				// generate a random profile name if not specified at first time
+				if profileName == "" {
+					profileName = namesgenerator.GetRandomName(0)
+				}
+			} else {
+				if profileName == "" {
+					profileName = configuration.DefaultProfile
```

Comment:
What does that mean? Why use a default name when configure a new profile?

Reply:
For updating default profile.

Reply:
Add a comment for this.